tutorials/RandomNumberGeneration/Tests.qs

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+//////////////////////////////////////////////////////////////////////
+// This file contains testing harness for all tasks.
+// You should not modify anything in this file.
+//////////////////////////////////////////////////////////////////////
+
+namespace Quantum.Kata.RandomNumberGeneration {
+    
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+    open Quantum.Kata.Utils;
+    open Microsoft.Quantum.Arrays;
+    
+
+    // Exercise 1.
+    operation T1_RandomBit_Test () : Unit {
+        Message("Testing...");
+        mutable count = 0;
```

Comment:
Running the exercise as it is given in the code stub (returning -1) gives a somewhat unexpected error "Unexpectedly low number of 1's generated. Generated -1000 out of 1000". I would add an extra check to verify that the returned numbers are always in the valid range.

Is there a way to reuse `CheckFlatDistribution` here as well?